### PR TITLE
Clean up code in prSignUp.css

### PR DESCRIPTION
**Pull request by Mannendri**
I cleaned up code in prSignUp.css, and adjusted the naming of a class (since it was in camelcase).

### CODE REVIEW

File: public/templates/prSignUp.html [hunk 1]
```diff
@@ -107,7 +107,7 @@
       <div class="tab-content">
         <!-- ***** Sign Up Form Start ***** -->
         <div id="signup">
-          <h1 id="signupText">Sign Up for Free</h1>
+          <h1 id="signup-text">Sign Up for Free</h1>
```
Comment (j0n3lson) [inline public/templates/prSignUp.html]: Seems like we need to have a convention here dashed or camel case. The line directly above it is inconsistent with this line `id="signup"` is neither camel or snake case. 

Maybe let's standardize this:
https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

Which says use-dashes as you have done. Could you fix this in this file?

**Review by j0n3lson — CHANGES_REQUESTED** [review]
Can you use the google style guide for all classes/ids in this file:

https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

Bonus points if you standardize this across the code base. :rocket: 

Addressing the full clean up in another PR.

**Comment by j0n3lson** [comment]
Friendly ping. Either a review is pending or changes are requested

File: public/templates/prSignUp.html (hunk 1 above)
Comment (Mannendri) [reply public/templates/prSignUp.html]: Yep!

**Comment by Mannendri** [comment]
I made a commit updating all classes and ids to match the Google Style Guide.

**Review by j0n3lson — APPROVED** [review]
Great job! You'll need to rebase for your other PR after I merge this.

Well done!

**Comment by Mannendri** [comment]
Hey @j0n3lson, I may be overthinking this, but how would I rebase for my other PR?

Would I simply follow the usual steps?: 
1) git fetch upstream
2) git checkout mainline
3) git merge upstream/mainline

**Comment by j0n3lson** [comment]
> Hey @j0n3lson, I may be overthinking this, but how would I rebase for my other PR?
> 
> Would I simply follow the usual steps?: 
> 1) git fetch upstream
> 2) git checkout mainline
> 3) git merge upstream/mainline

Note: You need to do these steps for your other PR. 

Yep. Same steps. Those three steps get you the updates `mainline` branch... You still need to update your branch (or rebase it). The last step is to checkout your branch and rebase it:

```
# Switch to your branch...
git checkout the-branch-name-here

# ...then stack the commits in this branch on
# top of the commits in mainline
git rebase mainline
```

**Comment by Mannendri** [comment]
Alright, did it!

**Comment by j0n3lson** [comment]
> Alright, did it!

Sweet! Let's get your other PR submitted!!!

**Pull request merged**